stream-api/src/dto/create-channel.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class CreateChannelDto {
+  englishName: string;
+  arabicName: string;
+  channelRoute: string;  
```

Comment:
whats this route?

Reply:
For creating new channels.

Reply:
every channel will have there unique route. 
For Example, Moulana Tariq Jameel route will be @MTJ.
https://ehsaan-stream.web.app/channels/@MTJ
We will call channels with this route. It is a unique column.

Reply:
ok this is the identifier or handle
not route. change to a different name